src/helm/benchmark/static/schema.yaml

```diff
@@ -1405,6 +1405,34 @@ metrics:
     description: The accuracy for the 10% of predictions that the model is most confident on (only computed for classification tasks).
     lower_is_better: false
 
+
+  # Fairness Metrics
+  - name: equalized_odds_difference
```

Comment:
Add these to `metric_groups` as well:
```
metric_groups:
  - name: decodingtrust_fairness_metrics
    display_name: Accuracy
    metrics:
      - name: equalized_odds_diference
        split: ${main_split}
      - name: odds-diff
        split: ${main_split}
```

Likewise for OOD knowledge metrics

Reply:
Thanks for the comment. We just added this in d0d031a5fcd11b77c698f899400b2d6355f1c219.